src/lib/command-helpers.mjs

```diff
@@ -151,7 +152,7 @@ export function isUnknownError(error) {
 export const validateDatabaseOrSecret = (argv) => {
   if (!argv.database && !argv.secret && !argv.local) {
     throw new ValidationError(
-      "No database or secret specified. Pass either --database, or --secret, or --local.",
+      "No database or secret specified. Please use either --database, --secret, or --local to connect to your desired Fauna database.",
```

Comment:
would it make sense for this function to return true or throw? then you could, e.g., directly return it in validate because it returns true.

Reply:
yaa good idea